src/action.rs

```diff
@@ -109,32 +109,66 @@ pub fn open_file(
     options: FileDialogOptions,
     file_info_action: impl Fn(Option<FileInfo>) + 'static,
 ) {
-    let send = create_ext_action(Scope::new(), move |path: Option<PathBuf>| {
-        file_info_action(path.map(|path| FileInfo { path, format: None }))
-    });
+    let send = create_ext_action(
+        Scope::new(),
+        move |(path, paths): (Option<PathBuf>, Option<Vec<PathBuf>>)| {
+            if paths.is_some() {
+                file_info_action(paths.map(|paths| FileInfo {
+                    path: paths,
+                    format: None,
+                }))
+            } else {
+                file_info_action(path.map(|path| FileInfo {
+                    path: vec![path],
```

Comment:
The trade off is we always return a Vec even when you can only select a single thing. I personally think that's easier to handle in user land but happy to hear others to chime in